tests/DocumentationTest.py

```diff
@@ -20,6 +20,7 @@
 from glob import glob
 
 class DocumentationTest(unittest.TestCase):
+    @unittest.expectedFailure
```

Comment:
Let's make the suggested changes and make a commit that gets rid of this so we know it will pass.